lib/va_profile/v2/contact_information/service.rb

```diff
@@ -216,12 +218,16 @@ def get_person_transaction_status(transaction_id)
 
         private
 
-        def icn_with_aaid
-          "#{@user.icn}^NI^200M^USVHA"
+        def verify_user!
+          unless @user&.vet360_id.present? || @user&.icn.present?
+            raise 'ContactInformationV2 - Missing User ICN and VAProfile_ID'
```

Comment:
The unless statement is an OR but then I see the error raise says AND "Missing User ICN **and** VAProfile_ID"

Reply:
Yes, the error will be raise if both are not present.